Engine/Source/Trail.cpp

```diff
@@ -0,0 +1,344 @@
+#include "Trail.h"
+#include "Application.h"
+#include "ModuleOpenGL.h"
+#include "ModuleResource.h"
+#include "Application.h"
+#include "ModuleOpenGL.h"
+#include "ModuleCamera.h"
+#include "CameraComponent.h"
+#include "ResourceTexture.h"
+#include "ModuleResource.h"
+#include "glew.h"
+#include "MathGeoLib.h"
+#include "Timer.h"
```

Comment:
#include "MathGeoLib.h"
is not needed

#include "ModuleResource.h"
#include "ModuleOpenGL.h"
#include "Application.h"
are duplicated